app/src/main/java/com/github/se/cyrcle/ui/review/AllReviewsScreen.kt

```diff
@@ -31,23 +32,38 @@ import androidx.compose.ui.unit.dp
 import com.github.se.cyrcle.R
 import com.github.se.cyrcle.model.parking.ParkingViewModel
 import com.github.se.cyrcle.model.review.ReviewViewModel
+import com.github.se.cyrcle.model.user.UserViewModel
 import com.github.se.cyrcle.ui.navigation.NavigationActions
+import com.github.se.cyrcle.ui.navigation.Screen
 import com.github.se.cyrcle.ui.theme.molecules.TopAppBar
+import com.google.firebase.Timestamp
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+fun Timestamp.toFormattedDate(): String {
+  val dateFormat = SimpleDateFormat("MMM dd, yyyy", Locale.getDefault())
```

Comment:
Typo. Also Timestamp.now() is doodoo i remember using it and it's pretty bad, small advice, dont rely on it, test it thoroughly !